boost.yml

```diff
@@ -0,0 +1,14 @@
+name: boost
+sources:
+  - type: archive
+    url: https://boostorg.jfrog.io/artifactory/main/release/1.74.0/source/boost_1_74_0.tar.bz2
+    sha256: 83bfc1507731a0906e387fc28b7ef5417d591429e51e788417fe9ff025e116b1
+    x-checker-data:
+      type: anitya
+      project-id: 6845
+      stable-only: true
+      url-template: https://boostorg.jfrog.io/artifactory/main/release/$version/source/boost_${major}_${minor}_$patch.tar.bz2
+buildsystem: simple
+build-commands:
+  - ./bootstrap.sh --prefix=/app
```

Comment:
Consider changing it to build&install only libraries which application uses.

Reply:
Sorry, I'm not familiar enough with Boost to understand this. What do I need to change?

Reply:
You can [take a look](https://www.google.com/search?q=site%3Agithub.com%2Fflathub+boost) what other Flatpaks use.

Reply:
This should give me a good idea
```
$ grep -rhi "#include <boost/" | sort | uniq
#include <boost/algorithm/string/predicate.hpp> 
#include <boost/config.hpp>                  
#include <boost/container/flat_map.hpp>                                                        
#include <boost/core/noncopyable.hpp>         
#include <boost/crc.hpp>                 
#include <boost/date_time/gregorian/greg_day.hpp>
#include <boost/date_time/posix_time/posix_time.hpp>
#include <boost/filesystem/fstream.hpp>
#include <boost/functional/hash.hpp>
#include <boost/lambda/lambda.hpp>             
#include <boost/lexical_cast.hpp>
#include <boost/log/attributes/clock.hpp>
#include <boost/log/attributes/counter.hpp>
#include <boost/log/attributes/current_process_id.hpp>
#include <boost/log/attributes/current_thread_id.hpp>
#include <boost/log/attributes.hpp>
#include <boost/log/attributes/mutable_constant.hpp>
#include <boost/log/common.hpp>
#include <boost/log/core.hpp>
#include <boost/log/expressions/formatters/date_time.hpp>
#include <boost/log/expressions.hpp>
#include <boost/log/expressions/keyword.hpp>
#include <boost/log/expressions/predicates/channel_severity_filter.hpp>
#include <boost/log/sinks/async_frontend.hpp>
#include <boost/log/sinks.hpp>
#include <boost/log/sinks/sync_frontend.hpp>
#include <boost/log/sinks/text_file_backend.hpp>
#include <boost/log/sinks/text_ostream_backend.hpp>
#include <boost/log/sources/global_logger_storage.hpp>
#include <boost/log/sources/record_ostream.hpp> 
#include <boost/log/sources/severity_channel_logger.hpp>
#include <boost/log/support/date_time.hpp>
#include <boost/log/trivial.hpp>
#include <boost/log/utility/exception_handler.hpp>
#include <boost/log/utility/manipulators/add_value.hpp>
#include <boost/log/utility/setup/console.hpp>
#include <boost/log/utility/setup/filter_parser.hpp>
#include <boost/log/utility/setup/from_stream.hpp>
#include <boost/log/utility/setup/settings.hpp> 
#include <boost/make_shared.hpp>
#include <boost/multi_index_container.hpp>
#include <boost/multi_index/hashed_index.hpp>
#include <boost/multi_index/key_extractors.hpp> 
#include <boost/multi_index/ordered_index.hpp>
#include <boost/range/iterator_range.hpp>
#include <boost/shared_ptr.hpp>
#include <boost/version.hpp>
```

Reply:
Done in 1e5643c